sdk/servicebus/service-bus/karma.conf.js

```diff
@@ -49,21 +49,21 @@ module.exports = function(config) {
     // https://www.npmjs.com/package/karma-env-preprocessor
     envPreprocessor: [
       "SERVICEBUS_CONNECTION_STRING",
-      "QUEUE_NAME",
-      "QUEUE_NAME_NO_PARTITION",
-      "QUEUE_NAME_SESSION",
-      "QUEUE_NAME_NO_PARTITION_SESSION",
-      "TOPIC_NAME",
-      "TOPIC_NAME_NO_PARTITION",
-      "TOPIC_NAME_SESSION",
-      "TOPIC_NAME_NO_PARTITION_SESSION",
-      "SUBSCRIPTION_NAME",
-      "SUBSCRIPTION_NAME_NO_PARTITION",
-      "SUBSCRIPTION_NAME_SESSION",
-      "SUBSCRIPTION_NAME_NO_PARTITION_SESSION",
-      "TOPIC_FILTER_NAME",
-      "TOPIC_FILTER_SUBSCRIPTION_NAME",
-      "TOPIC_FILTER_DEFAULT_SUBSCRIPTION_NAME",
+      "QUEUE_NAME_BROWSER",
+      "QUEUE_NAME_NO_PARTITION_BROWSER",
+      "QUEUE_NAME_SESSION_BROWSER",
+      "QUEUE_NAME_NO_PARTITION_SESSION_BROWSER",
+      "TOPIC_NAME_BROWSER",
+      "TOPIC_NAME_NO_PARTITION_BROWSER",
+      "TOPIC_NAME_SESSION_BROWSER",
+      "TOPIC_NAME_NO_PARTITION_SESSION_BROWSER",
+      "SUBSCRIPTION_NAME_BROWSER",
+      "SUBSCRIPTION_NAME_NO_PARTITION_BROWSER",
+      "SUBSCRIPTION_NAME_SESSION_BROWSER",
+      "SUBSCRIPTION_NAME_NO_PARTITION_SESSION_BROWSER",
+      "TOPIC_FILTER_NAME_BROWSER",
+      "TOPIC_FILTER_SUBSCRIPTION_NAME_BROWSER",
+      "TOPIC_FILTER_DEFAULT_SUBSCRIPTION_NAME_BROWSER",
```

Comment:
How are these values exactly? Does this meant that we can expect to see these env variables when running browser tests?

Reply:
Yes, these will be picked up from the .env file if user supplies them.